.azure-pipelines/pr_test_skip_scripts.yaml

```diff
@@ -7,6 +7,11 @@ t0:
   - k8s/test_config_reload.py
   - k8s/test_disable_flag.py
   - k8s/test_join_available_master.py
+  # Mclag test only support on t0-mclag platform which is not in PR test
+  - mclag/test_mclag_l3.py
+  # Nat feature is not enabled in current image
```

Comment:
Can you explain the reason more clearly?

Reply:
Modified